recipe/meta.yaml

```diff
@@ -19,16 +19,19 @@ requirements:
   host:
     - pip
     - python
-    - setuptools >=40.6.0
+    - setuptools
```

Comment:
setuptools and setuptools_scm can be removed as the build is now using hatchling
https://github.com/python-jsonschema/jsonschema/blob/v4.9.1/pyproject.toml#L2